emcli/commands/base.py

```diff
@@ -86,7 +86,7 @@ def match_command(cmd):
             (action, with_spinner) = command
             if with_spinner:
                 self.show_activity()
-            action(**self.cli_args)
+            return action(**self.cli_args)
```

Comment:
Execute the action that has been registered with the command and propagate the value back up the stack (to `__main__.main`)